Server/Plugins/APIDump/Hooks/OnMonsterMoved.lua

```diff
@@ -0,0 +1,27 @@
+return
+{
+	HOOK_MONSTER_MOVED =
+	{
+		CalledWhen = "After a monster has moved to a new position.",
+		DefaultFnName = "OnMonsterMoved",  -- also used as pagename
+		Desc = [[
+            This hook is called after a {{cMonster|monster}} has moved, and the
+            entity move packet has been broadcast to clients.</p>
```

Comment:
Where is the opening `<p>` ?

Reply:
The APIDump plugin inserts the initial and terminal `<p>`; if you want more paragraphs, only the inner ones are needed. This code is correct.

Reply:
But now we have two terminal `</p>` :confused: ? Just look at the [cCraftingGrid-Doc](https://api.cuberite.org/cCraftingGrid.html):

    <p>Also, an object of this type is used in <a
    href="cCraftingRecipe.html">cCraftingRecipe</a>'s ConsumeIngredients() function for
    specifying the exact number of ingredients to consume in that recipe; plugins may use this to
    apply the crafting recipe.</p></p>

If I understand it correctly, I would need to use it like this:

    Some Text</p><p>New Paragraph

But not at the end of the of the text!

Reply:
Right, the terminating `</p>` shouldn't be here.